out/build/x64-Debug/.cmake/api/v1/query/client-MicrosoftVS/query.json

```diff
@@ -0,0 +1 @@
+{"requests":[{"kind":"cache","version":2},{"kind":"cmakeFiles","version":1},{"kind":"codemodel","version":2}]}
```

Comment:
There's still accidentally commited files left in.

Reply:
Removed